exercise1/exercise2/main.cpp

```diff
@@ -0,0 +1,23 @@
+//Exercise 2 - Data Type Sizes
+Write a program that will output the size of the different data types in c++.
```

Comment:
Don't upload code that doesn't compile. This lines aren't commented, so you code should not be able to run